command/common/internal/help_all_display.go

```diff
@@ -64,7 +64,9 @@ var HelpCategoryList = []HelpCategory{
 	{
 		CategoryName: "ROUTES:",
 		CommandList: [][]string{
-			{"routes", "create-route", "check-route", "map-route", "unmap-route", "delete-route", "delete-orphaned-routes"},
+			{"routes", "route"},
+			{"create-route", "check-route", "map-route", "unmap-route", "delete-route"},
+			{"delete-orphaned-routes"},
```

Comment:
Why are we adding `create-route` and `delete-orphaned-routes` via this PR? 
cc: @bepotts @a-b